BoxContentSDK/BoxContentSDK/BOXURLSessionManager.h

```diff
@@ -81,14 +81,22 @@ totalBytesExpectedToSend:(int64_t)totalBytesExpectedToSend;
 @interface BOXURLSessionManager : NSObject
 
 /**
- * This method needs to be called once to set up the manager to be ready to perform other public methods
+ * Should only use sharedInstance instead of creating a new instance of BOXURLSessionManager
+ * BOXURLSessionManager is responsible for a unique background NSURLSession for the app
+ * with BOXURLSessionManager itself as delegate
+ */
++ (BOXURLSessionManager *)sharedInstance;
+
+/**
+ * This method needs to be called once to set up the manager to be ready to support background upload/download tasks.
```

Comment:
Should we support calling this more than once? Can the delegate theoretically change?